server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/EsMonitor.java

```diff
@@ -74,15 +68,6 @@ public int getNumberOfNodes() {
   @Override
   public Optional<Map<String, Object>> attributes() {
     Map<String, Object> attributes = new LinkedHashMap<>();
-
-    try (JmxConnection connection = jmxConnectionFactory.create(ProcessId.ELASTICSEARCH)) {
-      if (connection != null) {
-        EsSettingsMBean mbean = connection.getMBean(EsSettingsMBean.OBJECT_NAME, EsSettingsMBean.class);
-        attributes.put("Cluster Name", mbean.getClusterName());
```

Comment:
Nope, in fact it was out-of-scope of this ticket. There's currently no need for displaying cluster and node names. The only ES parameter that could be useful for debugging is the HTTP port.